noxfile.py

```diff
@@ -170,3 +172,73 @@ def docs_serve(session: Session) -> None:
         shutil.rmtree(build_dir)
 
     session.run("sphinx-autobuild", *args)
+
+
+@nox.parametrize("replay_file_path", glob.glob("./e2e-tests/cookiecutters/*.json"))
+@session(python=main_python_version)
+def test_cookiecutter(session: Session, replay_file_path) -> None:
+    """Uses the tap template to build an empty cookiecutter.
+
+    Runs the lint task on the created test project.
+    """
+    args = session.posargs or ["1"]
+
+    cc_build_path = "/tmp"  # noqa: S108
```

Comment:
I think you can use [`Session.create_tmp`](https://nox.thea.codes/en/stable/config.html#nox.sessions.Session.create_tmp) here. Wdyt?

Reply:
Thanks! But there is a little issue with build artifacts: we need to know which folder has the cookiecutters. 

Would  tempfile.gettempdir() work? Since it returns the name of the directory used for temporary files.